blueprint.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+  "meta": {
+    "title": "Remote Data Blocks latest",
+    "description": "Installs the latest release of remote-data-blocks plugin to WordPress Playground",
+    "author": "WordPress VIP",
+    "categories": [ "Content" ]
+  },
+  "features": {
+    "networking": true
+  },
+  "landingPage": "/wp-admin/",
+  "login": true,
+  "phpExtensionBundles": [ "light" ],
+  "preferredVersions": {
+    "php": "8.2",
+    "wp": "latest"
+  },
+  "steps": [
+    {
+      "step": "installPlugin",
+      "options": {
+        "activate": true
+      },
+      "pluginZipFile": {
+        "caption": "Installing Remote Data Blocks",
+        "resource": "url",
+        "url": "https://github-proxy.com/proxy/?repo=Automattic/remote-data-blocks&release=ae7f58e&asset=remote-data-blocks.zip"
```

Comment:
```suggestion
        "url": "https://github-proxy.com/proxy/?repo=Automattic/remote-data-blocks&release=latest&asset=remote-data-blocks.zip"
```

Reply:
I wish, doesn't work

Reply:
Hmm... thought @stoph was using that somewhere.

Reply:
@mehmoodak But is the downloaded file non-empty or unzippable? I think @stoph was working on a fix.

Reply:
It's empty 😓